<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientSetting</class>
 <widget class="QWidget" name="PatientSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>201</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>29</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>145</y>
     <width>149</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пол</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_2">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>230</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>259</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>201</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Цитологический диагноз</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>288</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>116</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>172</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>0</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>145</y>
     <width>108</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>М</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>58</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>259</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Комментарий</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>58</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>288</y>
     <width>107</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>87</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>29</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>172</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Гистологический диагноз</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>145</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>87</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>№ карты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>116</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата рождения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата заболевания</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
